include/bluetooth/scan.rst

```diff
@@ -58,45 +65,44 @@ Filters types
+|              | For example, several filters can be set for **name**, **address**, **UUID**, and **appearance**.          |
+|              | To trigger a **filter match** callback in this scenario, the following types have to match:               |
+|              |                                                                                                           |
+|              | * at least one of the address filters,                                                                    |
+|              | * at least one of the name filters,                                                                       |
+|              | * at least one of the appearance filters,                                                                 |
+|              | * all of the UUID filters.                                                                                |
+|              |                                                                                                           |
+|              | Otherwise, the **not found** callback is triggered.                                                       |
++--------------+-----------------------------------------------------------------------------------------------------------+
+
 
 Directed Advertising
 ====================
 
-To support receiving Directed Advertising packets with the Scanning Module, enable one of the following options in Zephyr:
+To receive directed advertising packets with the Scanning Module, enable one of the following options in Zephyr:
 
-* :option:`CONFIG_BT_PRIVACY` (scanning with changing addresses)
-* :option:`CONFIG_BT_SCAN_WITH_IDENTITY` (scanning with a local identity address)
+* :option:`CONFIG_BT_PRIVACY`  : scanning with changing addresses,
+* :option:`CONFIG_BT_SCAN_WITH_IDENTITY` : scanning with a local identity address.
 
-It is recommended to use the privacy option.
-When privacy is enabled directed advertising is only supported from bonded peers.
-Use scanning with identity only when scanning with privacy is not possible.
+It is recommended to enable the :option:`CONFIG_BT_PRIVACY` option to support directed advertising between bonded peers only.
+Use the :option:`CONFIG_BT_SCAN_WITH_IDENTITY` option only when the :option:`CONFIG_BT_PRIVACY` option is not feasible.
 
-When `Filters`_ are set, you can use the proprietary ``filter_no_match`` event to handle Directed Advertising.
-This event checks whether the Directed Advertising packets do not match any filters.
-In case of a positive verification, you can establish connection without the need to disable or reconfigure the existing filters.
+When `Filters`_ are set, you can use the proprietary ``filter_no_match`` event to handle directed advertising.
+This event checks whether the directed advertising packets do not match any filters.
+In case of a positive verification, where no matches are found, you can establish a connection without the need to disable or reconfigure the existing filters.
```

Comment:
Sentence "where no matches are found"  is a little confusing to me.

Reply:
Ditto, plus the line above. 

I think this part should be rewritten.

Reply:
Rephrased a bit, please check if it sounds better to you